src/cache.rs

```diff
@@ -16,6 +16,8 @@ use walkdir::{DirEntry, WalkDir};
 use crate::error::TealdeerError::{self, CacheError, UpdateError};
 use crate::types::{OsType, PathSource};
 
+static CACHE_DIR: &str = "TEALDEER_CACHE_DIR";
```

Comment:
I feel like this could be more descriptive, maybe something like `CACHE_DIR_ENV_VAR`?

Reply:
I like that you moved it to constant though! 👍

Reply:
Renamed.